src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -651,21 +651,29 @@ function disableRow(elem, row) {
     $tdElements.parent().hide();
 }
 
-function feedbackRecipientUpdateVisibilityOptions(elem) {
-    elem = $(elem);
-    if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
-        enableRow(elem, 1);
-        disableRow(elem, 3);
+function feedbackRecipientUpdateVisibilityOptions(recipientType) {
+    $recipientType = $(recipientType);
```

Comment:
As mentioned in the previous review, variables here should be local. <code>var ... = ...</code>